bat/oauth.yaml

```diff
@@ -57,7 +57,152 @@ paths:
                   grant_type: refresh_token
                   refresh_token: SqJDIwX00fehqHxS6xmb-kzqAlrYe_0EHgekMexVT8k
         description: ''
+  /catalog/account:
+    get:
+      description: Returns current user information
+      summary: Get an Account
+      tags:
+        - Account
+      operationId: Account Information
+      responses:
+        '200':
+          description: ok
+          content:
+            application/vnd.api+json:
+              schema:
+                $ref: '#/components/schemas/Account'
+        '401':
+          description: Unauthorised
+          content:
+            application/json:
+              schema:
+                $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-scale-api-definitions/master/common/CCS_Standard_API_Schema.yaml#/components/schemas/Errors'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-scale-api-definitions/master/common/CCS_Standard_API_Schema.yaml#/components/schemas/Errors'
+        '429':
+          description: Too Many Requests
+          content:
+            application/json:
+              schema:
+                $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-scale-api-definitions/master/common/CCS_Standard_API_Schema.yaml#/components/schemas/Errors'
+      security:
+        - bearerAuth: []
+    post:
+      description: >-
+        Creates a new account
+
+        This endpoint requires [Spree Auth Devise](https://github.com/spree/spree_auth_devise) gem installed
```

Comment:
Minor point - suggest maybe this implementation detail is not relevant in the context of our SCALE layer API

Reply:
Yes, true.
Will remove